src/LEDMatrixDriver.cpp

```diff
@@ -64,11 +64,11 @@ void LEDMatrixDriver::setPixel(int16_t x, int16_t y, bool enabled)
 
 bool LEDMatrixDriver::getPixel(int16_t x, int16_t y) const
 {
-	uint8_t* p = _getBufferPtr(x,(modRev) ? 7 - y : y);
+	uint8_t* p = _getBufferPtr(x,y);
 	if (!p)
 		return false;
 
-	uint16_t b = (modRev) ? (x & 7) : 7 - (x & 7);		//bit
+	uint16_t b = 7 - (x & 7);		//bit

```

Comment:
Do we still need to do this subtraction even if we have the proper rendering in `_displayRow()`?